cli/src/cmd/run.js

```diff
@@ -207,13 +208,21 @@ export async function runInDebug({
   spinner.start('Observing task...');
   const taskId = await iexec.deal.computeTaskId(dealid, 0);
   const taskObservable = await iexec.task.obsTask(taskId, { dealid: dealid });
+  const taskTimeoutWarning = setTimeout(() => {
+    spinner.warn('Task is taking longer than expected...');
+    spinner.start('Observing task...');
```

Comment:
Reset previous spinner text rather than hardcoding text (in this context, spinnerText contains logs from the container, starting with a hardcoded text would delete them)

```suggestion
    const spinnerText = spinner.text;
    spinner.warn('Task is taking longer than expected...');
    spinner.start(spinnerText); // restart spinning
```